src/exodus/hdmint/tracker.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019 The Zcoin Core Developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef EXODUS_HDMINT_TRACKER_H
+#define EXODUS_HDMINT_TRACKER_H
+
+#include "../../hdmint/mintpool.h"
+
+#include "hdmint.h"
+
+namespace exodus
+{
+
+class HDMintWallet;
+
+struct MintMeta
```

Comment:
After take a look in the code I think we should remove this. The reasons are:

1. It need to have duplicated fields with `HDMint`.
2. It just act like the lookup table to `HDMint`.

Instead of having a single map we can do multiple-map. With this way it will more faster.

Reply:
good idea. Then I change it to `boost::multi_index` instead.